load_data/import_gnomad_sv.py

```diff
@@ -0,0 +1,184 @@
+from gnomad_hail import *
+
+
+def import_vcf(overwrite: bool):
+    mt = hl.import_vcf(gnomad_sv_vcf_path, force_bgz=True, min_partitions=300)
+    meta = hl.import_table(gnomad_sv_release_samples_list_path, force=True, key='s')
+    mt = mt.annotate_cols(
+        release=hl.is_defined(meta[mt.col_key])
+    )
+    mt.write(gnomad_sv_mt_path, overwrite=overwrite)
+
+
+def merge_genomes_sv_samples(genomes_mt: hl.MatrixTable, sv_mt: hl.MatrixTable) -> hl.Table:
+    genomes_samples = genomes_mt.cols()
+
+    genomes_meta = get_gnomad_meta('genomes')
+    genomes_samples = genomes_samples.annotate(
+        **genomes_meta[genomes_samples.s].select('project_id', 'release')
+    )
+
+    genomes_samples = genomes_samples.key_by(
+        sv_s=(genomes_samples.project_id + "_" + genomes_samples.s).replace('\W+', '_')
+    )
+    genomes_samples = genomes_samples.select(
+        genomes_s=genomes_samples.s,
+        in_v2_genomes=True
+    )
+
+    gnomad_sv_samples = sv_mt.cols()
+
+    gnomad_sv_samples = gnomad_sv_samples.annotate(
+        in_mesa=gnomad_sv_samples.s.matches("G94818_.*"),
+        in_v2_sv=True
+    )
+
+    merged_samples = genomes_samples.join(gnomad_sv_samples, how='outer')
+    merged_samples = merged_samples.annotate(
+        in_v2_genomes=hl.is_defined(merged_samples.in_v2_genomes),
+        in_v2_sv=hl.is_defined(merged_samples.in_v2_sv)
+    ).persist()
+
+    print("Summary of sample sharing between gnomAD genomes and gnomad SVs:")
+    merged_samples.group_by(
+        'in_v2_genomes',
+        'in_v2_sv'
+    ).aggregate(
+        n=hl.agg.count(),
+        n_mesa=hl.agg.count_where(merged_samples.in_mesa)
+    ).show()
+
+    merged_samples = merged_samples.filter(
+        merged_samples.in_v2_genomes & merged_samples.in_v2_sv
+    )
+
+    return merged_samples
+
+
+def order_cols(mt1: hl.MatrixTable, mt2: hl.MatrixTable) -> Tuple[hl.MatrixTable, hl.MatrixTable]:
+    cols = mt1.add_col_index(name='idx1').cols().select('idx1')
+    cols = cols.join(mt2.add_col_index(name='idx2').cols().select('idx2'))
+    return (
+        mt1.choose_cols(cols.idx1.collect()),
+        mt2.choose_cols(cols.idx2.collect())
+    )
+
+
+def merge_with_short_variants(overwrite: bool, min_af: float):
+    # Load gnomAD genomes and filter to PASS / release sites and samples
+    gnomad_genomes = get_gnomad_data('genomes', adj=True, release_samples=True)
```

Comment:
We also have `release_annotations=True` so that you can skip the join in the next few lines